src/SFA.DAS.AdminService.Web/Views/Search/FrameworkReprintReason.cshtml

```diff
@@ -0,0 +1,142 @@
+﻿@model SFA.DAS.AdminService.Web.ViewModels.Search.FrameworkReprintReasonViewModel
+@using SFA.DAS.AdminService.Common.Extensions;
+@using SFA.DAS.AssessorService.Domain.Consts
+
+@{
+    ViewBag.Title = "Are you sure this certificate needs reprinting";
+    Layout = "_Layout";
+
+    bool isFirstResult = true;
+    var count = 0;
+}
+
+<a asp-action="FrameworkLearnerDetails" asp-controller="Search" class="govuk-back-link">Back</a>
+
+<main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <partial name="_ValidationSummaryPartial" />
+            <h1 class="govuk-heading-l">
+                @ViewBag.Title
+            </h1>
+            <dl class="govuk-summary-list">
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Certificate number
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.CertificateNumber
```

Comment:
This should be the Certificate Reference when it is present - it's not present now because this is the first reprint